couchbase/src/io/lcb/encode.rs

```diff
@@ -697,3 +698,23 @@ pub fn encode_kv_stats(instance: *mut lcb_INSTANCE, request: KvStatsRequest) {
         lcb_stats3(instance, cookie as *mut c_void, &command);
     }
 }
+
+/// Encodes a `PingRequest` into its libcouchbase `lcb_CMDPING` representation.
+pub fn encode_ping(instance: *mut lcb_INSTANCE, request: PingRequest) {
+    let cookie = Box::into_raw(Box::new(request.sender));
+
+    let report_id = match request.options.report_id {
```

Comment:
This can be simplified to something like unwrap_or() on the optional.. if you are worried about the uuid allocation, there is also a variant IIRC which takes a callback